EnglishFilter/sections/specific.txt

```diff
@@ -272,6 +272,8 @@ musixmatch.com##.lyrics-bottom-ad-container
 /duplichecker.com\/\w{4}\/side_bar\./$domain=duplichecker.com
 /duplichecker.com\/\w{4}\/\w\d\./$domain=duplichecker.com
 cointelegraph.com##.banner-blocked
+ownedcore.com##a[href^="https://brobot.me/"][target="_blank"]
+ownedcore.com###sidebar > li > div.smaller:has(> div.blocksubhead > span:contains(Sponsored Ads))
```

Comment:
@stanislav-atr Please add `?` to this rule -> `ownedcore.com#?##sidebar > li > div.smaller:has(> div.blocksubhead > span:contains(Sponsored Ads))` and move it to `css_extended.txt`